src/functions.rs

```diff
@@ -973,7 +969,7 @@ fn call_to_fsc_rules<'a>(
                     context_range: context_str_arg
                         .get_range()
                         .ok_or_else(|| CascadeErrors::from(InternalError::new()))?,
-                    file: file.clone(),
```

Comment:
The internet is not being super helpful, what is the diffrence between clone and cloned?

Reply:
Clone would clone the iterator itself, while cloned is a function on the iterator which clones each element as it iterates:

https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.cloned